data/core/dirwatch.lua

```diff
@@ -171,10 +171,10 @@ end
 
 
 -- compute a file's info entry completed with "filename" to be used
--- in project scan or falsy if it shouldn't appear in the list.
+-- in project scan or return false if it shouldn't appear in the list.
```

Comment:
I think `falsy` is more correct as this can return `nil`. @adamharrison can you confirm?

Reply:
``get_project_file_info`` can return nil, yes, if not both `info` and `info.type` are valid